plugins/modules/web_infrastructure/bugzilla.py

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Chuang Cao <chcao@redhat.com>
+# Atlassian open-source approval reference OSR-76.
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
```

Comment:
This can be removed. It is no longer needed (and eventually will lead to an error).